src/products/Product.java

```diff
@@ -81,14 +84,7 @@ public int getQuantity() {
 
     @Override
     public String toString() {
-        return "Product{" +
-                "brand=" + brand +
-                ", color='" + color + '\'' +
-                ", price=" + price +
-                ", description='" + description + '\'' +
-                ", name='" + name + '\'' +
-                ", quantity=" + quantity +
-                '}';
+        return "Product{" + "brand=" + brand + ", color='" + color + '\'' + ", price=" + price + ", description='" + description + '\'' + ", name='" + name + '\'' + ", quantity=" + quantity + '}';
```

Comment:
è ancora disordinato

Reply:
fai in modo che stampi ogni variabile per ogni riga con relativo valore e togli le parentesi